app/assets/stylesheets/map/leaflet.css.scss

```diff
@@ -245,6 +245,33 @@
 	border: 5px solid #bbb;
 	}
 
+.leaflet-draw-section {
+	position: relative;
+	top: 110px;

```

Comment:
The motivation was the zoom in/out buttons move down for the "create dataset from query" banner, but the draw buttons don't (They're inserted into the page by Leaflet differently). This height has them lined up when the banner is present, moving them up would make it so the banner obscures them. I figured styling wasn't a priority so long as the functionality exists so I didn't investigate further.